tasks/configure-agent.yml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ---
-- name: Configure the agent with the Stackdriver API key.
+- name: Configure the agent.
   template: src=stackdriver.sysconfig dest={{ stackdriver_sysconfig }} backup=yes
   notify:
```

Comment:
Indeed, you are right. I finally changed the name in configure-agent.yml to be more explicit for gcm users.